### PR TITLE
added handlers.py script

### PR DESCRIPTION
**Pull request by AnushaKonduru**

### CODE REVIEW

File: logagg/collect/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+### Install the `logagg` package from
+```bash
+https://github.com/deep-compute/logagg.git
+cd logagg
+sudo python setup.py install
+```
+
+### Usage
+
+- After installation the logagg module, We can perform opertions in python shell.
```
Comment (venugopalkandru) [inline logagg/collect/README.md]: @AnushaKonduru @praneeth36 Did we pass sentences through Grammarly?
If not please do, thank you.

File: logagg/collect/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+nginx: configuration file /etc/nginx/nginx.conf test is successful
+```
+
+- Restart the nginx by using command:
+```bash
+/etc/init.d/nginx restart
+```
+
+
```
Comment (venugopalkandru) [inline logagg/collect/README.md]: @AnushaKonduru @praneeth36 We don't need more than one new line to separate out sections.

File: logagg/collect/README.md (hunk 1 above)
Comment (venugopalkandru) [reply logagg/collect/README.md]: @AnushaKonduru @praneeth36 Wanted you to check all sentences you guys used in the entire code base to pass through Grammarly.

**Comment by venugopalkandru** [comment]
@praneeth36 @AnushaKonduru For now merging this pull request, though some modifications are required and @AnushaKonduru is not available to fix those.
Please fix this after merging this pull request.

**Comment by praneeth36** [comment]
Sure @venugopalkandru

**Pull request merged**